circle.yml

```diff
@@ -535,7 +535,7 @@ jobs:
             ~/tests/EIPTests/BlockchainTests/
       - download_execution_tests:
           repo: ipsilon/tests
-          rev: eof-relaxed-stack-validation-20240221
+          rev: create3-not-supported
```

Comment:
I took the branch form https://github.com/ethereum/tests/pull/1337 and rolled back some changes there for supporting truncated data in containers. In this PR truncated data is not supported yet.